bin/postman-test/collections/system-management.postman_collection.json

```diff
@@ -0,0 +1,990 @@
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "health",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "07675d9c-ca27-4208-b998-81ae040e36f5",
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Response time is less than 600ms\", function () {",
+							"    pm.expect(pm.response.responseTime).to.be.below(600);",
+							"});",
+							"",
+							"pm.test(\"Content-Type is application/json\", function(){",
+							"    var contentType = pm.response.headers.get('Content-Type');",
+							"    pm.expect(contentType).to.include('application/json');",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "api-key",
+						"value": "{{api-key}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{systemManagementUrl}}/api/v1/health/edgex-core-command,edgex-core-data,edgex-core-metadata,edgex-export-client,edgex-export-distro,edgex-support-logging,edgex-support-notifications,edgex-support-scheduler",
+					"protocol": "http",
+					"host": [
+						"localhost"
+					],
+					"port": "48090",
+					"path": [
+						"api",
+						"v1",
+						"health",
+						"edgex-core-command,edgex-core-data,edgex-core-metadata,edgex-export-client,edgex-export-distro,edgex-support-logging,edgex-support-notifications,edgex-support-scheduler"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "ping",
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "api-key",
+						"value": "{{api-key}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{systemManagementUrl}}/api/v1/ping",
+					"protocol": "http",
+					"host": [
+						"localhost"
+					],
+					"port": "48090",
+					"path": [
+						"api",
+						"v1",
+						"ping"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "metrics",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "ed75887a-472a-4f12-857a-1d790af5ab0a",
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Response time is less than 600ms\", function () {",
+							"    pm.expect(pm.response.responseTime).to.be.below(600);",
+							"});",
+							"",
+							"pm.test(\"Content-Type is application/json\", function(){",
+							"    var contentType = pm.response.headers.get('Content-Type');",
+							"    pm.expect(contentType).to.include('application/json');",
+							"});",
+							""
```

Comment:
You can add assertions to validate the response content which contains the required information.
For example,  [this script](https://github.com/edgexfoundry/blackbox-testing/blob/master/bin/postman-test/collections/device-virtual.postman_collection.json#L99)  checks the name is correct or existed

Reply:
- I have added an inline code-comment in this PR: Please confirm whether the assertions I've added (such as the ones mentioned inline, to validate the response content which contains the required information) align with the blackbox testing approach.
- If so, I will add further assertions: Note that I have added a set of assertions as an example for you to examine 👍
- I looked up [some Postman resources](https://learning.getpostman.com/docs/postman/scripts/test_examples/) but they were for simple JSON responses, and not nested JSON responses. Thus, the following did not work:
```
tests["Status code is 200"] = responseCode.code === 200;
  if(responseCode.code === 200){
      tests["Contains Configuration"] = responseBody.has("Configuration");
      }
```
- So please check out my new set of assertions, which I've provided as an example. If good, I will add further assertions 👍

Reply:
We could use new style test script syntax instead of the old. Please refer to https://learning.getpostman.com/docs/postman/scripts/test_examples/

And I found some usages to test nested JSON responses.
```
....
pm.test("Content-Type is application/json", function(){
    var contentType = pm.response.headers.get('Content-Type');
    pm.expect(contentType).to.include('application/json');
});

pm.test("Response must contains required data", function () {
    var jsonData = pm.response.json();
    pm.expect(jsonData).to.have.property("Configuration");
    pm.expect(jsonData.Configuration).to.have.property("edgex-core-command");
    pm.expect(jsonData.Configuration).to.have.property("edgex-core-data");
   ...
});
```
Then we can write the assertion to expect the result is correct.

https://learning.getpostman.com/docs/postman/scripts/postman_sandbox_api_reference/#pmcookies
https://www.chaijs.com/api/bdd/

Reply:
That sounds good: I will follow up on making use of the new-style test script syntax you mention, and update this PR accordingly 👍

Reply:
- This work has now been completed.
- Commits have been squashed and build checks running.
- Requesting a final review, thank you 👍